enhancements/microshift/microshift-updateability-ostree.md

```diff
@@ -427,122 +419,79 @@ from clean state.
-   > so only the latter is considered.
-
-**If _current boot's commit_ and _previous boot's commit_ are the same<a name="current-previous-same"></a>**
-
-It means that current commit booted more than once in a row.
-
-1. If backup for _current boot's commit_ exists: **restore**
-   > This means that if backup exists it is always restored regardless of the
-   > system's health (except when it was unknown)
-   > ([see optional restore after backing up](#current-commit-different-restore)).
-   >
-   > - If this is greenboot's reboot, then backup was created before this commit was staged and deployed.
-   >   So commit must've been reintroduced and [data would be restored early in the process](#current-commit-different-restore).
-   >   - Being in this place in decision tree means that either restore failed or system was unhealthy.
-   >     Restoring again seems to best bet.
-   >
-   > - If this is manual reboot, then system was healthy,
-   >   then manually rebooted, backed data up and end up unhealthy,
-   >   again rebooted resulting in being in this place in decision tree.
-   >   - Admin should address problems before rebooting the system
-   >     (and retrigger greenboot first, to refresh system's health)
-   >   - Restoring data seems okay - going back to last healthy state.
-
-1. If backup for _current boot's commit_ does not exist
-   > There's no proof of the commit being ever healthy
-
-   1. If "history of commits" only knows _current boot's commit_: **delete data and [start cluster](#starting-the-cluster)**
-      > First commit with MicroShift running on the system, system is consistently unhealthy.
-
-   1. If "history of commits" knows about _previous commit with MicroShift_
-      > MicroShift was already running on the device.
-      > But it doesn't mean system will rollback to that _previous commit with MicroShift_
-
-      1. _Previous commit with MicroShift_ is the same as _rollback_
-         1. If version metadata matches _previous commit with MicroShift_: **backup data and proceed to [data migration](#data-migration-1)**
-            > Previous boot of current commit might have been unhealthy because it failed to make a backup.
-
-         1. Otherwise: **restore backup of _previous commit with MicroShift_**
-            > Give chance to migrate data and start cluster again.
-            > Assumption that admin upgraded from healthy system is important here.
-
-      1. _Previous commit with MicroShift_ is not the _rollback_ **delete data and [start cluster](#starting-the-cluster)**
-         > Means that rollback does not feature MicroShift.
-         > This is "retry boot" of FDO scenario.
-
-##### Data migration
-
-1. Compare version persisted in metadata with MicroShift's binary
+   **Assume it's 4.13, back up the data and proceed with startup**
+
+1. If previous boot was healthy: **backup data for previous boot**
+
+1. If current commit is different from previous boot's commit,
```

Comment:
Same question on the term `commit`, since we change the phrase `ostree commit` to `ostree deployment`, does that apply here as well?